pkgs/applications/science/math/caffe/default.nix

```diff
@@ -13,8 +13,8 @@
 , Accelerate, CoreGraphics, CoreVideo
 , lmdbSupport ? true, lmdb
 , leveldbSupport ? true, leveldb, snappy
-, cudaSupport ? stdenv.isLinux, cudatoolkit
-, cudnnSupport ? false, cudnn ? null
+, cudaSupport ? config.cudaSupport or false, cudatoolkit
+, cudnnSupport ? cudaSupport, cudnn ? null
```

Comment:
Do you want to change the default on Linux here? Given it is Deep Learning, running it CPU-only might be too painful to be useful…